game/src/js/sw.js

```diff
@@ -1,23 +1,17 @@
 /* global Promise */
-const STATIC_CACHE = 'gosch-static-cache-v2';
-const ASSETS_CACHE = 'gosch-assets-cache-v2';
-const CACHES = [
-  STATIC_CACHE,
-  ASSETS_CACHE
-];
+const RM_CACHE = 'raginmagin-cache-v4';
 
 self.addEventListener('install', e => {
   e.waitUntil(
     caches
-      .open(STATIC_CACHE)
+      .open(RM_CACHE)
       .then(cache => {
         fetch('../../assets/assets.json').then(function(response) {
           return response.json();
         }).catch(function(err) {
           console.log('fetch:', err);
         }).then(function(files) {
```

Comment:
Maybe `files` here should be named something more descriptive like `assets` or `assetManifest`.

Reply:
okay I'll change that.